Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -181,15 +181,7 @@ + (NSInteger)getRetryCount {
 }
 
 + (NSString *)getAppKey {
-    NSString *ret = [[[NSBundle mainBundle] infoDictionary] objectForKey:KEY_APP_KEY];
-    if (!ret || ret.length == 0) {
-        // for backward compatibility
-        ret = [BNCPreferenceHelper readStringFromDefaults:KEY_APP_KEY];
```

Comment:
I don't think so -- if you recall, we decided that we needed to stop pulling these items from defaults as it could cause unexpected behavior when a key is (perhaps accidentally) removed, and the app keeps working.

For instance, say I used key "asdf" at some point, which was then stored into defaults (I don't think anything stores into defaults for key anymore, but say in a past version it did). If I then strip out that line of code and release, the app will incidentally continue to work for users who already had the app, but won't work for new users.

I'm open to discussion on this, but I'm of the opinion that we shouldn't be pulling from there.